includes/woopay/class-woopay-session.php

```diff
@@ -277,6 +285,196 @@ private static function get_payload_from_cart_token() {
 		return null;
 	}
 
+	/**
+	 * Returns the encrypted session request for the frontend.
+	 *
+	 * @return array The encrypted session request or an empty array if the server is not eligible for encryption.
+	 */
+	public static function get_frontend_init_session_request() {
+		if ( ! WC_Payments_Features::is_client_secret_encryption_eligible() ) {
+			return [];
+		}
+
+		$session = self::get_init_session_request();
+
+		$store_blog_token = Jetpack_Options::get_option( 'blog_token' );
```

Comment:
Maybe we can change this line to:

```php
$store_blog_token = WC_Payments::mode()->is_dev() ? 'dev_mode' : Jetpack_Options::get_option( 'blog_token' );
```

To avoid using the same condition on line `308` and `311`.